content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,348 @@
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+   The command output should be similar to this example:
+
+   {{< code text >}}
+root@sensu00:~# etcdctl snapshot --endpoints https://sensu-backend-01:2379 save sensu_etcd_snapshot.db
+{"level":"info","ts":"2022-08-18T20:47:28.419Z","caller":"snapshot/v3_snapshot.go:65","msg":"created temporary db file","path":"sensu_etcd_snapshot.db.part"}
+{"level":"info","ts":"2022-08-18T20:47:28.452Z","logger":"client","caller":"v3/maintenance.go:211","msg":"opened snapshot stream; downloading"}
+{"level":"info","ts":"2022-08-18T20:47:28.452Z","caller":"snapshot/v3_snapshot.go:73","msg":"fetching snapshot","endpoint":"https://sensu-backend-01:2379"}
+{"level":"info","ts":"2022-08-18T20:47:28.512Z","logger":"client","caller":"v3/maintenance.go:219","msg":"completed snapshot read; closing"}
+{"level":"info","ts":"2022-08-18T20:47:28.648Z","caller":"snapshot/v3_snapshot.go:88","msg":"fetched snapshot","endpoint":"https://sensu-backend-01:2379","size":"2.9 MB","took":"now"}
+{"level":"info","ts":"2022-08-18T20:47:28.648Z","caller":"snapshot/v3_snapshot.go:97","msg":"saved","path":"sensu_etcd_snapshot.db"}
+Snapshot saved at sensu_etcd_snapshot.db
+{{< /code >}}
+
+2. Start a new Sensu backend or cluster and confirm that it is running:
+
+   {{< language-toggle >}}
+
+{{< code shell "Single backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380 \
+--etcd-initial-cluster-token sensu-backend-01 \
+--etcd-initial-advertise-peer-urls http://localhost:2380 \
+--etcd-advertise-client-urls http://localhost:2379
+{{< /code >}}
+
+{{< code shell "Clustered backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-01 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--etcd-advertise-client-urls http://sensu-backend-01:2379
+
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-02 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-02:2380 \
+--etcd-advertise-client-urls http://sensu-backend-02:2379
+
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-03 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--etcd-advertise-client-urls http://sensu-backend-03:2379
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+3. Copy the etcd snapshot file to each cluster member so that all members will be restored using the same snapshot.
```

Comment:
I'm not sure I've explained this adequately. I don't completely understand what "copy to each cluster member" means. Is there a separate directory location for each backend?

Reply:
There is. For each backend, there is a data-dir. By default, it's `/var/lib/sensu/sensu-backend`